extensions/default/src/CustomizeableContextMenu/types.ts

```diff
@@ -0,0 +1,123 @@
+import { Types } from '@ohif/core';
+
+/**
+ * SelectorProps are properties used to decide whether to select a manu or
+ * menu item for display.
+ * An instance of SelectorProps is provided to the selector functions, which
+ * return true to include the item or false to exclude it.
+ * The point of this is to allow more specific conext menus which hide
+ * non-relevant menu options, optimizing the speed of selection of menus
+ * (See Bill Wallace's masters thesis for selection time versus complexity of user menus).
```

Comment:
That's awesome. However, as charming and intriguing as this is, consider excluding it OR if you have a link perhaps keep this and include the link.

Reply:
I was wondering if anyone would comment - have removed it :-)